keps/sig-node/2400-node-swap/README.md

```diff
@@ -1,4 +1,4 @@
-# KEP-2400: Node system swap support
+# KEP-2400: Node system swap enablement
```

Comment:
```suggestion
# KEP-2400: Node memory swap support
```
The authoritative title is the title of the KEP issue.

Reply:
The same here.